src/CMakeLists.txt

```diff
@@ -352,6 +352,14 @@ set(mmapper_SRCS
     proxy/proxy.h
     proxy/telnetfilter.cpp
     proxy/telnetfilter.h
+    roompanel/CRoom.cpp
```

Comment:
The prefix of "C" is a PandoraMapper naming anti-pattern. We're starting to converge on CamelCase naming convention for files

Reply:
fixed